### PR TITLE
Update version release to v3.3.5

### PR DESCRIPTION
**Pull request by mattymaxwell**

### CODE REVIEW

**Review by RobjS — COMMENTED** [review]
@mattymaxwell The changelog seems to be listing 3.3.4 twice?

```
Unreleased
Removed
Support for PHP versions below 7.4
[3.3.4] - 2022-02-15
Changed
permission to allow Editors access to menus and widgets
[3.3.4] - 2022-02-15
Changed
Mustache bumped to v2.14.1 to patch [security issue](https://github.com/advisories/GHSA-4rmr-c2jx-vx27)
```

(from https://github.com/dxw/govuk-blogs/blob/028615273b1065a35c8787b8c1a1d16d12f4c225/CHANGELOG.md)

**Comment by mattymaxwell** [comment]
> @mattymaxwell The changelog seems to be listing 3.3.4 twice?
> 
> ```
> Unreleased
> Removed
> Support for PHP versions below 7.4
> [3.3.4] - 2022-02-15
> Changed
> permission to allow Editors access to menus and widgets
> [3.3.4] - 2022-02-15
> Changed
> Mustache bumped to v2.14.1 to patch [security issue](https://github.com/advisories/GHSA-4rmr-c2jx-vx27)
> ```
> 
> (from https://github.com/dxw/govuk-blogs/blob/028615273b1065a35c8787b8c1a1d16d12f4c225/CHANGELOG.md)

Thanks @RobjS have updated that to what it should be now

**Comment by RobjS** [comment]
@mattymaxwell The removal of PHP versions below 7.4 will be part of this release as well. Each release will include any changes that have been merged into `master` since the previous release - there's no way of doing "selective" releases.

**Comment by mattymaxwell** [comment]
@RobjS ah sorry I didn't think to bring those above the line in. Done now

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.3.5] - 2022-10-10
+
 ## Unreleased
```
[the remark this answers is not in the document]
Comment (mattymaxwell) [reply CHANGELOG.md]: @RobjS that's removed now

**Review by RobjS — APPROVED** [review]

**Pull request merged**